### DISCUSSION TITLE
Question about having multiple google accounts linked

### DISCUSSION THREAD

**Discussion by Lan-Phan** [ory/kratos#3780] [Q&A] [answered]
Hello,

I'd like to implement 1 authentication system which allow users to link multiple accounts of same Oauth2 server (for ex, Google) to their own account, such as:
+ Their own account is ABC
+ They enable Google Oauth2 provider
+ They can link their personal google account like abc1@gmail.com and abc2@gmail.com into ABC
+ They can have SSO via abc1@gmail.com and abc2@gmail.com

Is it possible to do that via Ory ecosystem: kratos / hydra / oathkeeper?

**Comment by jonas-jonas** [comment, marked as answer]
Yes, this should be possible using Ory Kratos' OIDC strategy: https://www.ory.sh/docs/kratos/social-signin/overview

**Reply by Lan-Phan** [reply]
Great,

Thanks so much, @jonas-jonas 
Close this.
